erts/configure.in

```diff
@@ -990,6 +990,7 @@ case $host_os in
 		LDFLAGS="$save_ldflags"
 	;;
 esac
+AC_MSG_CHECKING(for final extra flags needed to export symbols)
```

Comment:
Otherwise we produce an isolated `none`. I'm just not sure whether this should be chained with the previous validation or have a different string, here.

Reply:
I don't understand the need for this. Can you show an example where this is needed?

Reply:
There's no "need" 😄, it's just that this produces an isolated `none` in a given line. This seems to pertain to nothing (it also doesn't help parse the logs). I'll run `configure` again and show you the output.

Reply:
I'm not sure about the message I chose, though, since I know not about initial expectations.

Reply:
This is the surrounding output (context):
```
checking how to create a directory including parents... /usr/local/opt/coreutils/libexec/gnubin/install -c -d
checking for extra flags needed to export symbols... none
none
checking for sin in -lm... yes
checking for dlopen in -ldl... yes
```

I can revert that change if you prefer.

Reply:
ah, ok. Now I understand. The fix would be to remove the AC_MSG_RESULT at line [989](https://github.com/erlang/otp/pull/2777/files/5a05fbd602103864f68994ab90bba05ba96cae46#diff-900a5d0495caac38a1f51a2cae1b2594R989).

Reply:
(I knew _something_ had to be fixed, but wasn't exactly sure what 😄)

Reply:
There's these 3 other occurrences (which I find strange, but probably are on purpose) in the log:
```erlang
checking for zlib inflateGetDictionary presence... checking for library containing inflateGetDictionary... no
checking for debug build of wxWidgets... checking for wx-config... /usr/local/bin/wx-config
checking for standard build of wxWidgets... checking for wx-config... (cached) /usr/local/bin/wx-config
```
(repeated `checking` with no line break)

[I'm running `./configure` with no options, here]

Reply:
b2338eb: rebased off `maint` and squashed for this discussion thread.

Reply:
This should fix those.

```
diff --git a/erts/configure.in b/erts/configure.in
index c616f9a926..8bc7446a65 100644
--- a/erts/configure.in
+++ b/erts/configure.in
@@ -1365,7 +1365,6 @@ error
 ])
 
 if test "$Z_LIB" != ""; then
-  AC_MSG_CHECKING(for zlib inflateGetDictionary presence)
   AC_SEARCH_LIBS(inflateGetDictionary, [z],
                  AC_DEFINE(HAVE_ZLIB_INFLATEGETDICTIONARY, 1,
                            [Define if your zlib version defines inflateGetDictionary.]))
diff --git a/lib/wx/configure.in b/lib/wx/configure.in
index 5a69738544..3ce422d89d 100644
--- a/lib/wx/configure.in
+++ b/lib/wx/configure.in
@@ -335,9 +335,15 @@ elif test  X"$MIXED_VC" = X"no"; then
     # Try to find debug libs first 
     # wxelibs=core,base,gl,aui,adv
 
+    AM_PATH_WXCONFIG($reqwx, wxWinWithGLDBG=1, wxWinWithGLDBG=0, [stc,xrc,html,adv,xml,core,base,gl,aui], [--unicode --debug=yes])
+
     AC_MSG_CHECKING(for debug build of wxWidgets)
+    if test "$wxWinWithGLDBG" = 1; then
+      AC_MSG_RESULT(yes);
+    else
+      AC_MSG_RESULT(no);
+    fi
 
-    AM_PATH_WXCONFIG($reqwx, wxWinWithGLDBG=1, wxWinWithGLDBG=0, [stc,xrc,html,adv,xml,core,base,gl,aui], [--unicode --debug=yes])
     DEBUG_WX_CFLAGS=$WX_CFLAGS
     DEBUG_WX_CXXFLAGS=$WX_CXXFLAGS
     DEBUG_WX_LIBS=$WX_LIBS
@@ -347,9 +353,15 @@ elif test  X"$MIXED_VC" = X"no"; then
     AC_SUBST(DEBUG_WX_LIBS)
     AC_SUBST(DEBUG_WX_LIBS_STATIC)
     
-    AC_MSG_CHECKING(for standard build of wxWidgets)
     AM_PATH_WXCONFIG($reqwx, wxWinWithGL=1, wxWinWithGL=0, [stc,xrc,html,adv,xml,core,base,gl,aui], [--unicode --debug=no])
 
+    AC_MSG_CHECKING(for standard build of wxWidgets)
+    if test "$wxWinWithGL" = 1; then
+      AC_MSG_RESULT(yes);
+    else
+      AC_MSG_RESULT(no);
+    fi
+
     if test "x$WX_LIBS_STATIC" = "x"; then
        WX_HAVE_STATIC_LIBS=false
     else
```

Reply:
7e49eac: hope you don't mind having pushed with your name and e-mail.